.github/workflows/pr.yml

```diff
@@ -0,0 +1,9 @@
+name: CI (PR)
```

Comment:
could we give this a different workflow name. There is an action called `pages-build-deployment` which is run when a gh-pages PR is merged so maybe we could call it `pages CI PR` ?